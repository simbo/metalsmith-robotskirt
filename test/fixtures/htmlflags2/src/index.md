
# Further Tests

## The `SAFELINK` Flag

[a safe link](http://domain.tld)

[an "unsafe" link](ssh://domain.tld)

## XHTML Conformance

This line break â†’  
should be XHTML-conformant.
