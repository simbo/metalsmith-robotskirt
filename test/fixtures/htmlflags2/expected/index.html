<h1 id="toc_0">Further Tests</h1>

<h2 id="toc_1">The <code>SAFELINK</code> Flag</h2>

<p><a href="http://domain.tld">a safe link</a></p>

<p>[an &ldquo;unsafe&rdquo; link](ssh://domain.tld)</p>

<h2 id="toc_2">XHTML Conformance</h2>

<p>This line break â†’<br/>
should be XHTML-conformant.</p>
